include/ofi_mem.h

```diff
@@ -602,6 +602,15 @@ static inline void *ofi_ibuf_alloc_at(struct ofi_bufpool *pool, size_t index)
 	return buf;
 }
 
+/*
+ * Empty buf pool init function that can be used to avoid initializing
+ * newly allocated buf pool entries.
+ */
+static inline void ofi_bufpool_init_none(struct ofi_bufpool_region *region, void *buf)
+{
+	/* This function is purposely empty */
+}
+
```

Comment:
This won't be a static inline function.  The function is passed by referenced, so actually needs to live somewhere.  I'd call this 'noop' instead of 'none'.